lib/data/live_matches_fixtures.freezed.dart

```diff
@@ -306,6 +306,150 @@ abstract class _MatchEvents implements MatchEvents {
       throw _privateConstructorUsedError;
 }
 
+MatchLineUps _$MatchLineUpsFromJson(Map<String, dynamic> json) {
```

Comment:
You should put freezed files in .gitignore to avoid pushing this much code. Leave those that are already on the repository. Everyone can later run `dart run build_runner` if needed